### ISSUE TITLE
Please support RDP 

### ISSUE THREAD

**Issue by Mannshoch** [EXALAB/AnLinux-App#462]
Please support RDP. 
RDP has more feature like dynamic screen resolution, directory and usb device share.

Currently, there is the problem that on fdroid there is no rdp client.

**Comment by Mannshoch** [comment]
As an alternative for RDP may Gamestream ([Sunshine](https://github.com/LizardByte/Sunshine) Server / [Moonlight](https://github.com/moonlight-stream/moonlight-qt) Client) could be a valid possibility. 

Both have the possibility to get the clients Screen resolution and HDI Interfaces and stream them back to the server for a seamless work.

Together with Android [Desktop-Mode](https://discord.com/channels/870877411049357352/870877411623985154/1326881354629255198) It would be a game changer.

**Comment by yeet-dot-org** [comment]
> Please support RDP. RDP has more feature like dynamic screen resolution, directory and usb device share.
> 
> Currently, there is the problem that on fdroid there is no rdp client.

Agreed could be very useful. Maybe sunshine or apollo would be even better.
